README.md

```diff
@@ -4,7 +4,7 @@
 
 The proto files can be consumed as GIT submodules or copied and built directly in the consumer project.
 
-The compiled files are published to central repositories (Maven, NPM...) from OpenTelemetry client libraries.
+The compiled files are published to central repositories (Maven, ...) from OpenTelemetry client libraries.
```

Comment:
Not sure if it is publish to maven, I think they no longer do that. @jkwatson ?

Reply:
They are still published to maven via [opentelemetry-proto-java](https://github.com/open-telemetry/opentelemetry-proto-java).